File: templates/go/main_test.go
package main

import (
	"strings"
	"testing"
)

func TestPartOne(t *testing.T) {
	input := strings.Split(`{{INPUT_1}}`, "\n")
	expected := {{ANSWER_1}}

	output := PartOne(input)

	if output != expected {
		t.Errorf("Output is %v, expeced %v", output, expected)
	}
}

//**scaf**|func TestPartTwo(t *testing.T) {
//**scaf**|	input := strings.Split(`{{INPUT_2}}`, "\n")
//**scaf**|	expected := {{ANSWER_2}} 
//**scaf**|
//**scaf**|	output := PartTwo(input)
//**scaf**|
//**scaf**|	if output != expected {
//**scaf**|		t.Errorf("Output is %v, expeced %v", output, expected)
//**scaf**|	}
//**scaf**|}

File: templates/ts/main.test.ts
import {describe, it, expect} from "vitest"
import { partOne } from "./main";
//**scaf**|import { partTwo } from "./main";

describe("Task {{DAY}}/{{YEAR}}", () => {
  it("test part one", () => {
    const input = `{{INPUT_1}}`.split("\n");

    expect(partOne(input)).toBe({{ANSWER_1}});
  });

  //**scaf**|it("test part twp", () => {
  //**scaf**|  const input = `{{INPUT_2}}`.split("\n");
  //**scaf**|
  //**scaf**|  expect(partTwo(input)).toBe({{ANSWER_2}});
  //**scaf**|});
});
